radar-upload-frontend/src/components/Patient/PatientTable/Records.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <v-list
+    shaped
+    three-line
+    subheader
+  >
+    <v-layout justify-center>
+      <v-progress-circular
+        class="mt-2"
+        v-show="loading"
+        indeterminate
+        color="primary"
+      />
+      <v-alert
+        type="error"
+        v-text="error"
+        v-show="error"
+        dense
+        text
+      />
+    </v-layout>
+    <v-subheader>
+      <span v-show="!loading">
+        <slot name="fileListSubHeader" />
+      </span>
+      <span v-show="!loading&&patientRecords.length==0">
+        This patient does not have any records
+      </span>
+    </v-subheader>
+    <v-list-group
+      v-for="record in records"
+      :key="record.id"
+      v-model="record.active"
+      no-action
+    >
+      <template #activator>
+        <v-list-item-avatar>
+          <!-- <v-progress-circular
+            v-if="record.status!=='READY'"
+            indeterminate
+            color="primary"
+          /> -->
+          <v-icon
+            color="info"
+            v-if="record.status==='READY'"
+          >
+            mdi-checkbox-marked-circle
+          </v-icon>
+        </v-list-item-avatar>
+
+        <v-list-item-content>
+          <v-list-item-title>
+            Records ID:
+            {{ record.id }} ({{ record.sourceType }})
+          </v-list-item-title>
+          <v-list-item-subtitle>
+            {{ record.message }}
+          </v-list-item-subtitle>
```

Comment:
Can we also list the `record.status`?

Reply:
I replace the status with an icon. You may see the icon display with condition of status in the code. Later when the flow is more clear, I will put on more icon and maybe with the tooltip

Reply:
I suggest to show the status with an icon. It is quite an important part of the work-flow. Right now, we have more or less a stable work-flow and we should have a front-end that can work with current work-flow.